package.json

```diff
@@ -47,6 +47,7 @@
   },
   "devDependencies": {
     "@babel/types": "^7.16.0",
+    "@types/cheerio": "^0.22.31",
```

Comment:
Typescript wouldn't compile without having the cheerio types due to the import in html.ts, interested to know how you've been getting around this?

Reply:
Not sure actually, I guess I must have `npm install`ed them at some point